### ISSUE TITLE
F2 key is not working, when open IDA with non-English input method

### ISSUE THREAD

**Issue by Jck574** [fjh658/IDA7.0_SP#2]
I used the lib that you've compiled ,and now my IDA stop crashing.
But I found there is still another problem, 
when I open IDA with non-English input methods, the F2 key can't work.
I thought it is a similar problem, can you fix it?

**Comment by fjh658** [comment]
You are right. This is a bug in Qt5.6, ida 7.0 is based on Qt 5.6.  There are a lot of shortcuts that can't be used, under non-English input methods. For example, tab, you need to press fn+tab. F2 is a function key. When fn+F2, it will become other meanings. Qt5.9 has fixed this bug. Backporting will have some cost.  @Jck574

**Comment by fjh658** [comment]
1. Unzip libqcocoa.zip, replace it.
2. In the non-English input method to start ida, the first time you need to switch to the English input method, you can use F2. After that, the input method can be switched casually.

```
md5 libqcocoa.zip
MD5 (libqcocoa.zip) = eb0b9db69c17d7a7439f7a71d722f594
```
[libqcocoa.zip](https://github.com/fjh658/IDA7.0_SP/files/2483294/libqcocoa.zip)

@Jck574 try it.

**Comment by fjh658** [comment]
Please use the latest version, it is already supported.  @Jck574

**Comment by Jck574** [comment]
Didn't expect you to fix it so quickly, thank you very much !!!

**Comment by fjh658** [comment]
Please close this issue. @Jck574 Thank you.
